Doc/tools/extensions/pyspecific.py

```diff
@@ -78,13 +80,12 @@ class ImplementationDetail(Directive):
     final_argument_whitespace = True
 
     # This text is copied to templates/dummy.html
```

Comment:
Was it copied in order to be picked up by translation system?  If yes, then it should be removed now (I am assuming that the new system based on `_` function works like classic gettext extraction-then-runtime-lookup)

Reply:
This message was already copied into `tools/templates/dummy.html`. Nothing changed.
https://github.com/python/cpython/blob/master/Doc/tools/templates/dummy.html

So far, Sphinx does not support translation in 3rd party extensions. At that time, this extension uses a bit hacky solution to realize that. But the hack does not work since Sphinx-1.8 because the structure of Sphinx was changed to provide i18n APIs for extensions.

This patch only enables to translate messages in extensions. And this does not change the script for message extraction. IMO, it would be better to fix the translation at first. So I'll post another change to extract messages from source code in the next step.